keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -138,7 +195,13 @@ to implement this enhancement.
 
 
 ##### Unit tests
-
+There will addition of new tests and modification of existing ones in the following packages:
```

Comment:
can you grab the current % coverage values for those directories, as described in the KEP template